pkg/nsx/services/vpc/vpc.go

```diff
@@ -12,7 +12,7 @@ import (
 
 var (
 	log             = logger.Log
-	ResourceTypeVPC = common.ResourceTypeVPC
+	ResourceTypeVPC = common.ResourceTypeVpc
```

Comment:
do we need a new var, or why not just directly use common.ResourceTypeVpc as resource type where necessary?